### ISSUE TITLE
learning rate per subject ?

### ISSUE THREAD

**Issue by 1blackbar** [ShivamShrirao/diffusers#80]
**Is your feature request related to a problem? Please describe.**
When you train multiple subjects at once even with same amount of pics like 3 in my example , some subjects are trained faster than others and they overfit faster , how to solve that ?
**Describe the solution you'd like**
maybe learning rate per subject but this would be hard to predict how its overfitting, maybe a code to detect how much it overfit and reduce the rate for this subject ?
Im training multiple subjects and this is going to be an issue , its harder to put them on other backgrounds/enviroments before likeness is learned.Or is the 3 img per subject the problem here ?
Also shouldnt the rate be 1e-7 like actual training of SD  with laion ?

![image](https://user-images.githubusercontent.com/101844463/198012564-3be063ce-b8d1-495b-8c38-ab12ba385a4b.png)
![image](https://user-images.githubusercontent.com/101844463/198012674-73ab92c3-0dd8-47c8-b12a-ceb86a8213d1.png)
![image](https://user-images.githubusercontent.com/101844463/198012930-ab9701e1-c6a8-4299-9961-b529fd1db017.png)
